source/tests/system/nirfmxnr_driver_api_tests.cpp

```diff
@@ -68,12 +61,55 @@ class NiRFmxNRDriverApiTests : public Test {
   std::unique_ptr<NiRFmxNR::Stub> stub_;
 };
 
+#define CHECK_ERROR(session) if (1) \
```

Comment:
+1 for implementing CHECK/EXPECT methods these as macros. I haven't been doing that, but any time you can put an assert wrapper in a macro, it preserves the line number of the caller. That helps a ton with debugging failures.

Reply:
Another +1 for the macros! I was wondering what the advantage of them were at first but that's really handy.